### PR TITLE
Editorial updates on column IDs and Display Names

### PR DESCRIPTION
**Pull request by jpradocueva**
These changes won't affect the content in Release V1.1. They are actually editorial guidelines according to the working group agreement.

### CODE REVIEW

**Comment by jpradocueva** [comment]
This issue was originated during the discussion of #469

File: EDITORIAL_GUIDELINES.md [hunk 1]
```diff
@@ -130,6 +130,11 @@ These guidelines can be modified through a Pull Request (PR), which the members
 
 * **Normative Requirements as a Bullet List**: Normative statements (those using Normative Keywords) should be written as bullet points instead of lengthy sentences. 
 
+* **Column IDs:** They SHOULD be used in normative text sections, such as when specifying mandatory rules, schema definitions, or other implementation-related content. These Column IDs should be formatted without spaces and should match the exact naming conventions used in the schema (e.g., CommitmentDiscountID). 
+
+* **Display Names:** They SHOULD be used in introductory or explanatory sections where natural language context is more appropriate. In these sections, display names should follow normal text conventions, including spaces between words (e.g., Commitment Discount ID).
```
Comment (ijurica) [inline EDITORIAL_GUIDELINES.md:135]: I suggest we mention that the Display Names are also used in the Usability Constraints section (assuming we'll approve and merge the PR #559)

**Comment by jpradocueva** [comment]
### Action Items from TF-2 call on October 3rd:
- [ ] [TF2-[#582](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/issues/582)] Members: Coordinate with the editorial team to finalize minor changes by Monday.

File: EDITORIAL_GUIDELINES.md (hunk 1 above)
Comment (jpradocueva) [reply EDITORIAL_GUIDELINES.md:135]: @ijurica. do you think that it is necessary to indicate any particular section beyond, say, any introductory or explanatory section?

Comment (ijurica) [reply EDITORIAL_GUIDELINES.md:135]: you are right. there is no need to emphasize it. 
on that note, I'm resolving this conversation.

**Comment by jpradocueva** [comment]
### Action Items assigned during TF-2 call on Oct 15:
-  [ ] [TF-1-[#582](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/pull/582)] Joaquin, @jpradocueva: Fix double colon issues in Column IDs and obtain final approval.

**Comment by jpradocueva** [comment]
@ijurica, I suggest that you add this extra row to the table so it is easy to review. If the group disagrees with the change, it can be removed.

**Comment by ijurica** [comment]
Nit: add `<br>` in line 60

File: EDITORIAL_GUIDELINES.md [hunk 2]
```diff
@@ -41,28 +39,44 @@ These guidelines can be modified through a Pull Request (PR), which the members
...
+        </td>
+        <td>
+            This column:<br>
+            &nbsp;&nbsp; * MUST be null when ChargeCategory is "Tax" ...
+        </td>
+        <td>
+            - Enclosed in double quotation marks<br>
+            - Normal text without bold or italics
+        </td>
+    </tr>
```
Comment (ijurica) [inline EDITORIAL_GUIDELINES.md:79]: **Current status**: 
While we have consistently formatted normalized dimensions values as normal text without bold or italics, enclosed in double quotation marks, for metrics and non-normalized dimensions values, we've been using various formatting., such as:

- "TB"
- `Hours`
- Compute Usage

**Note/Question:** We internally use the terms normalized and non-normalized dimensions, but we haven't defined them. Should we?

---

OPTION B:

```suggestion
    <tr>
        <td><strong>Normalized Dimension Values:</strong></td>
        <td>
            - "Usage"<br>
            - "Tax"<br>
        </td>
        <td>
            This column:<br>
            &nbsp;&nbsp; * MUST be null when ChargeCategory is "Tax" ...
        </td>
        <td>
            - Enclosed in double quotation marks<br>
            - Normal text without bold or italics
        </td>
    </tr>
    <tr>
        <td><strong>Metric &amp; Non-normalized Dimension Values:</strong></td>
        <td>
            - `Hours`<br>
            - `TB`<br>
            - `Compute Usage`
        </td>
        <td>
            Common examples include the number of hours for compute appliance runtime (e.g. `Hours`) ...
        </td>
        <td>
            - Monospaced text, enclosed in backticks, without bold or italics<br>
        </td>
    </tr>
```

Comment (jpradocueva) [reply EDITORIAL_GUIDELINES.md:79]: The group agreed to go with option A.

**Comment by jpradocueva** [comment]
Approved during the Members' call on Oct 17.

**Pull request merged**